app/views/api/v2/models/main.json.rabl

```diff
@@ -3,3 +3,11 @@ object @model
 extends "api/v2/models/base"
 
 attributes :info, :created_at, :updated_at, :vendor_class, :hardware_model
+
+if params.has_key?(:include_permissions)
```

Comment:
this is make sense to extract it to a common view? (assuming it will be used elsewhere?)

Reply:
Sure. Make sense.